self-hosted/upgrades/upgrade-docker.md

```diff
@@ -35,123 +35,204 @@ mounts, or bind mounts.
     ```
+    This returns either `volume` or `bind`.
 
-1.  Get the current name or mount path with this command, and record it to use
-    when you perform the upgrade. Make sure you copy the correct command, based
-    on your mount point type.
+1.  Note the volume or bind used by your container:
 
     <Terminal>
 
-    <tab label='Volume mount'>
+    <tab label='Volume'>
 
     ```bash
     docker inspect timescaledb --format='{{range .Mounts }}{{.Name}}{{end}}'
+    ```
+    Docker returns the `<volume ID>`. You see something like this: 
+
+    ```
     069ba64815f0c26783b81a5f0ca813227fde8491f429cf77ed9a5ae3536c0b2c
     ```
 
     </tab>
 
-    <tab label='Bind mount'>
+    <tab label='bind'>
 
     ```bash
     docker inspect timescaledb --format='{{range .Mounts }}{{.Source}}{{end}}'
+    ```
+
+    Docker returns the `<bind path>`. You see something like this: 
+
+    ```
     /path/to/data
     ```
 
     </tab>
 
     </Terminal>
 
+    You use this value when you perform the upgrade.
+
 </Procedure>
 
-## Upgrade TimescaleDB within Docker
+## Upgrade $TIMESCALE_DB within Docker
 
-To upgrade TimescaleDB within Docker, you need to download the upgraded image,
+To upgrade $TIMESCALE_DB within Docker, you need to download the upgraded image,
 stop the old container, and launch the new container pointing to your existing
 data.
 
+<Tabs label="Upgrade TimescaleDB in Docker">
+
+<Tab title="TimescaleDB-HA">
+
 <Procedure>
 
-### Upgrading TimescaleDB within Docker
+1.  **Pull the latest $TIMESCALE_DB image**
 
-1.  Pull the latest TimescaleDB image. This command pulls the image for
-    TimescaleDB 2.17.x running on $PG 17. If you're using another $PG version, 
-    look for the relevant tag in the
-    [TimescaleDB HA Docker Hub repository](https://hub.docker.com/r/timescale/timescaledb-ha/tags).
+    This command pulls the latest version of $TIMESCALE_DB running on $PG 17:
 
-    ```bash
+    ```
     docker pull timescale/timescaledb-ha:pg17
     ```
 
-1.  Stop the old container, and remove it:
+    If you're using another version of $PG, look for the relevant tag in the [$TIMESCALE_DB HA](https://hub.docker.com/r/timescale/timescaledb-ha/tags) repository on Docker Hub.
+
+1.  **Stop the old container, and remove it**
 
     ```bash
     docker stop timescaledb
     docker rm timescaledb
     ```
 
-1.  Launch a new container with the upgraded Docker image, pointing to the
-    existing mount point. Make sure you copy the correct command, based on your
-    mount point type.
+1. **Launch a new container with the upgraded Docker image**
 
-    For volume mounts:
+   Launch based on your mount point type:
 
-    <Terminal>
+   <Terminal>
 
     <tab label='Volume mount'>
 
     ```bash
-    docker run -v 069ba64815f0c26783b81a5f0ca813227fde8491f429cf77ed9a5ae3536c0b2c:/var/lib/postgresql/data \
-      -d --name timescaledb -p 5432:5432 timescale/timescaledb-ha
+    docker run -v <volume ID>:/home/postgres/pgdata/data
```

Comment:
again, needs PGDATA and don't want to tell people to use /home/postgrs/pgdata/data

Reply:
Done